src/components/pages/PointShare/RenderPointShare.js

```diff
@@ -1,34 +1,80 @@
+/*
+This component contains temporary hard coding to fill data that should be pulling from the database.
```

Comment:
Comments added to aid future development and necessary future hard code removal for this component.